app/assets/stylesheets/alchemy/_extends.scss

```diff
@@ -31,15 +31,28 @@
   line-height: $form-field-line-height;
   transition: $transition-duration;
 
-  &:focus {
+  &:focus:not(.readonly) {
```

Comment:
Since it's not part of this commit I assume there is an existent `.readonly` class being used but it just has no styling applied to it, is that right?

Reply:
The class is used in the next commit ;)

Reply:
https://github.com/AlchemyCMS/alchemy_cms/pull/2167/commits/0a4b4073da35f01db62162fdfdeeb2b43d0f11b2#diff-e6722cb4a4b8150bc2d7f4b3b64eba928901c492461bc09ad3aac283ebfb7ae2R7

Reply:
Never mind, I just saw the next commit adding `readonly`.

Reply:
haha, commenting in parallel :) Github is becoming a chat :)